src/screens/ChannelListScreenV2/IncognitoEmptyChat/index.tsx

```diff
@@ -0,0 +1,119 @@
+        <TouchableOpacity style={styles.item}>
+          <View style={styles.itemContent}>
+            <Text style={styles.itemTitle}>Start an Incognito Chat</Text>
+            <Text style={styles.itemSubtitle}>Send an incognito message to friends.</Text>
+          </View>
+          <View>
+            <MemoIc_arrow_right width={8} height={12} fill={COLORS.gray400} />
+          </View>
+        </TouchableOpacity>
+        <TouchableOpacity style={styles.item}>
+          <View style={styles.itemContent}>
+            <Text style={styles.itemTitle}>Create an Incognito Post</Text>
+            <Text style={styles.itemSubtitle}>Post and comment using an alias.</Text>
+          </View>
+          <View>
+            <MemoIc_arrow_right width={8} height={12} fill={COLORS.gray400} />
+          </View>
+        </TouchableOpacity>
+        <TouchableOpacity style={styles.item}>
+          <View style={styles.itemContent}>
+            <Text style={styles.itemTitle}>Join Communities in Incognito Mode</Text>
+            <Text style={styles.itemSubtitle}>Safely join without anyone knowing.</Text>
+          </View>
+          <View>
+            <MemoIc_arrow_right width={8} height={12} fill={COLORS.gray400} />
+          </View>
+        </TouchableOpacity>
+      </View>
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  container: {
+    position: 'absolute',
+    width: '100%',
+    bottom: 0,
+    paddingTop: dimen.normalizeDimen(140),
+    paddingBottom: dimen.normalizeDimen(40),
+    justifyContent: 'flex-end',
+    alignItems: 'center',
+    zIndex: 10
+  },
+  image: {
+    width: widthScreen - 90,
+    height: 160,
+    resizeMode: 'contain'
+  },
+  titleText: {
+    fontFamily: fonts.inter[600],
+    fontSize: normalizeFontSize(16),
+    lineHeight: normalizeFontSize(24),
+    color: COLORS.anon_primary,
+    paddingHorizontal: dimen.normalizeDimen(40),
+    textAlign: 'center',
+    marginTop: dimen.normalizeDimen(16)
+  },
+  wrapper: {
+    paddingHorizontal: dimen.normalizeDimen(20),
+    width: '100%'
+  },
+  line: {
+    height: 2,
+    backgroundColor: COLORS.gray200,
+    marginBottom: dimen.normalizeDimen(24)
+  },
+  item: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    backgroundColor: COLORS.white,
+    paddingHorizontal: dimen.normalizeDimen(16),
+    paddingVertical: dimen.normalizeDimen(15),
+    borderRadius: 8,
+    borderColor: COLORS.gray200,
+    borderWidth: 1,
+    marginBottom: dimen.normalizeDimen(8)
+  },
+  itemContent: {
+    flex: 1
+  },
+  itemTitle: {
+    fontFamily: fonts.inter[600],
+    fontSize: normalizeFontSize(14),
+    lineHeight: normalizeFontSize(18),
+    color: COLORS.anon_primary,
+    marginBottom: dimen.normalizeDimen(4)
+  },
+  itemSubtitle: {
+    fontFamily: fonts.inter[400],
+    fontSize: normalizeFontSize(14),
+    lineHeight: normalizeFontSize(18),
+    color: COLORS.gray500
+  }
+});
+
+export default IncognitoEmptyChat;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `IncognitoEmptyChat` component seems to be well implemented. However, it's missing any form of user interaction handling on the `TouchableOpacity` components (lines 24, 33, and 42). If these are meant to trigger some action when pressed, you should add an `onPress` prop to handle this.

```diff
        <TouchableOpacity style={styles.item} onPress={handleStartIncognitoChat}>
...
        <TouchableOpacity style={styles.item} onPress={handleCreateIncognitoPost}>
...
        <TouchableOpacity style={styles.item} onPress={handleJoinCommunitiesInIncognitoMode}>
```

You would then define these handler functions within your component:

```javascript
const handleStartIncognitoChat = () => {
  // logic for starting an incognito chat
};

const handleCreateIncognitoPost = () => {
  // logic for creating an incognito post
};

const handleJoinCommunitiesInIncognitoMode = () => {
  // logic for joining communities in incognito mode
};
```

Remember to ensure that these handlers are properly tested and that they perform the expected actions.


<!-- This is an auto-generated comment by OSS CodeRabbit -->